mm2src/coins/eth/v2_activation.rs

```diff
@@ -916,3 +924,50 @@ fn compress_public_key(uncompressed: H520) -> MmResult<H264, EthActivationV2Erro
     let compressed = public_key.serialize();
     Ok(H264::from(compressed))
 }
+
+#[cfg(not(target_arch = "wasm32"))]
+async fn run_db_migration_for_new_eth_pubkey(ctx: &MmArc, keypair: &KeyPair) -> MmResult<(), EthActivationV2Error> {
+    let db_id = hex::encode(dhash160(keypair.public().as_bytes()));
+    let shared_db_id = shared_db_id_from_seed(&db_id)
```

Comment:
Question:
In my understanding we always can have only single shared_db_id for the whole app and it is derived from the passphrase. Is this intended to get this shared_db_id from hash160 of pubkey?

Reply:
we can't generate `shared_db_id` from the passphrase using `shared_db_id_from_see` so we need to decide on how to generate `shared_db_id` for pubkeys when in non `Iguana mode`

Reply:
why do we need a `shared_db` when we are doing `eth pubkey` activation?

Reply:
eth uses HD mode too